addons/dragging/functions/fnc_carryObject.sqf

```diff
@@ -19,6 +19,11 @@
 params ["_unit", "_target"];
 TRACE_2("params",_unit,_target);
 
+// If in ViV cargo, unload it first
+if (!isNull isVehicleCargo _target) then {
+    objNull setVehicleCargo _target;
```

Comment:
could get return value for errors, but I have no idea why it would fail
```suggestion
    if (!(objNull setVehicleCargo _target)) then { WARNING_1("ViV Unload Failed %1",_target); };
```